app/controllers/resources_controller.rb

```diff
@@ -1,8 +1,60 @@
 class ResourcesController < ApplicationController
-  before_action :logged_in_user, only: :index
+  before_action :logged_in_user, only: %i(create new index)
+  before_action :validate_create_form, only: :create
+
   def index
     @projects = Project.filter_name(params[:name])
                        .filter_date(params[:date])
     @pagy, @projects = pagy @projects, items: Settings.pagy.number_items
   end
+
+  def new
+    @project_user = ProjectUser.new
+  end
+
+  def create
+    date = Date.parse params.dig(:project_user, :joined_at)
+    @month = date.month
+    @year = date.year
+    @project_id = params.dig(:project_user, :project_id)
+    @list_params = params.dig(:project_user, :project_user_resources_attributes)
+
+    return unless create_resources
+
+    flash[:success] = t(".create_success")
+    redirect_to resources_path
+  end
+
+  private
+
+  def validate_create_form
+    return if params.dig(:project_user,
+                         :project_user_resources_attributes).present?
+
+    flash[:danger] = t ".at_least_1"
+    redirect_to new_resource_path
+  end
+
+  def create_resources
+    @list_params.each do |_key, resource_params|
+      user_id = resource_params["project_user_id"].to_i
+      percentage = resource_params["percentage"].to_i
+      man_month = percentage.to_f / 100.0
+      project_user = ProjectUser.find_by(user_id:,
+                                         project_id: @project_id)
```

Comment:
Nếu chỗ này, em chỉ check sự tồn tại thôi thì cân nhắc dùng code where exists? chẳng ạn 
`@project.project_users.exists?(user_id: user_id)

Reply:
https://stackoverflow.com/questions/23857857/passing-two-parameters-to-exist-in-rails

Reply:
em còn lấy id của nó nữa ạ